python_modules/dagster-graphql/dagster_graphql/schema/inputs.py

```diff
@@ -80,6 +81,7 @@ def to_selector(self):
             updated_after=updated_after,
             created_before=created_before,
             created_after=created_after,
+            exclude_subruns=self.excludeSubruns,
```

Comment:
showBackfillRunsAsSingleRun?
aggregateBackfillRuns?

I feel like we should be explicit about what it is rather than making a new term ("subruns")

Reply:
got it. In that case we should have a tooltip to explain explicitly what sub runs are. Currently they are runs launched by a backfill only right?